sorter.go

```diff
@@ -63,6 +65,9 @@ func (l keyList) Less(i, j int) bool {
 		if ai != bi {
 			return ai < bi
 		}
+		if ar[i] == br[i] {
+			continue
```

Comment:
@rogpeppe is there a chance that this would be fixed anytime soon, it really hurts us on some scenarios